services/rclone-http/default.nix

```diff
@@ -9,7 +9,6 @@
 with lib;
 let
   cfg = config.services.rclone-http;
-  rclone = pkgs.callPackage ../../pkgs/rclone { };
```

Comment:
Upstream rclone >= v1.68.0 should have all patches we need and have. `pkgs/rclone/*` can be removed, it only contained the override and vendored patches.

Reply:
Ah, I see, we call this explicitly elsewhere, ugh.

I'd like to see this cleaned up in the future. Can you provide some more detailed logs how exactly this broke the uploads?

Reply:
Yes we still need the old patched rclone in the jenkins controller. Unfortunately the log I shared is the only error I got out of jenkins so I have no idea what is going wrong there.

This can be reproduced by running any pipeline in a jenkins controller instance that has rclone >= 1.68.0 installed. The archival step will fail.